app/scripts/utils/dict-from-tuples.js

```diff
@@ -1,15 +1,23 @@
+// @ts-check
+
 /**
- * Create a dictionary from a list of [key,value] pairs.
- * @param tuples: A list of [key,value] pairs
- * @return: A dictionary
+ * Create a dictionary from a list of [key, value] pairs.
+ *
+ * TODO(Trevor): Replace with Object.fromEntries?
```

Comment:
Think we could probably replace these implementations with aliases for:

- `dictItems`: `Object.entries`
- `dictKeys`: `Object.keys`
- `dictValues`: `Object.values`